qa/tasks/cephfs/test_volumes.py

```diff
@@ -201,8 +201,13 @@ def _verify_clone(self, subvolume, snapshot, clone,
         self._verify_clone_root(path1, path2, clone, clone_group, clone_pool)
         self._verify_clone_attrs(path1, path2)
 
-    def _gen_name(self, name, n):
-        names = [f'{name}{random.randrange(0, 9999)}{i}' for i in range(n)]
+    def _gen_name(self, name, n=1):
```

Comment:
Let me answer that based on the commit message:

> 2. Don't produce a new random number everytime, rather let the name be a
   series. This is more predictable and helps while debugging tests.

This could be somewhat helpful, but really, you'd need to focus on what in the test went wrong rather than how the test presents itself upon failure. Moreover, in real world, failures would not show up in a way that makes it easy to debug. A few extra minutes here and there to debug a test is not going to get any negative attention. 

> 2. Set default value for "n", the parameter through which we tell the
   number of names to be generated.

QA code need not be perfect. It's totally fine to pass in the value each time.

> 3. No need to loop when "n=1", return random generated name directly.

I bet python optimizes this for us.

>5. Starting numbering the names from one. This way last digit of name is
   1 for first name.

How is this making debugging easier?

Reply:
> you'd need to focus on what in the test went wrong rather than how the test presents itself upon failure.

That helps understanding and fixing the failure. Which saves times and more importantly the effort required at every instance of that failure.

The original point was not even presentation. The point of this commit was to have predictable names for clones so that I can cancel them manually when cloner threads crashes due to a bug and get test running in shortest time. This was helpful and since it was small change I kept it. :)

Anyways, removed.

Reply:
> > you'd need to focus on what in the test went wrong rather than how the test presents itself upon failure.
> 
> That helps understanding and fixing the failure. Which saves times and more importantly the effort required at every instance of that failure.
> 
> The original point was not even presentation. The point of this commit was to have predictable names for clones so that I can cancel them manually

A manual step that's required during development isn't a reason to do such changes. Imagine this (bug in cloner thread) happening in the a real world cluster - would it be sensible to recommend the cluster operator to use predictable clone names so that it does not take much thinking to cancel them?